src/components/calculator.js

```diff
@@ -1,36 +1,61 @@
 import React, { Component } from 'react';
+import InputButton from './input';
+import calculate from '../logic/calculate';
 import '../index.css';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
```

Comment:
- [ ] Multiplication operation isn't working please investigate this issue.
![image](https://user-images.githubusercontent.com/33767951/145474209-6a3eabc4-5ae1-4ae7-b700-ade861d3fd73.png)